pdoc/__init__.py

```diff
@@ -199,6 +199,8 @@ class GoldenRetriever(Dog):
 You can find an example at
 [`examples/custom-template/module.html.jinja2`](https://github.com/mitmproxy/pdoc/blob/main/examples/custom-template/module.html.jinja2).
 
+Hiding an item only removes it from documentation. It is still displayed in the source code when clicking the "View Source" button.
```

Comment:
Happy with the wording; I'd suggest moving it up one paragraph though, before 'As a last resort..' but not critical. Thanks.